src/components/Data/Layout.vue

```diff
@@ -72,7 +72,7 @@ export default {
             collection: this.$route.params.collection
           })
         })
-        .catch(err => console.error(err))
+        .catch(() => null)
```

Comment:
Is it OK to hide errors to the user?

Reply:
Error thrown here are Unauthorized errors for list collections on restricted indexes

Reply:
How can you be 100% sure the _only_ errors thrown here are related to rights?

Reply:
Fixed